crates/spirv-std/src/lib.rs

```diff
@@ -116,6 +116,13 @@ pub use runtime_array::*;
 #[cfg(feature = "glam")]
 pub use glam;
 
+// HACK(shesp) As we removed support for generic user-configurable vector types in the Image API,
+// glam is now required. In the future we might want to add other popular vector math libraries,
+// so we keep the "glam" feature toggle intact for now. As the code won't currently compile
+// without it, present the user with a somewhat friendly error message.
+#[cfg(not(feature = "glam"))]
+compile_error!("`spriv-std` now requires the use of `glam`. Make sure the "glam" feature is specified for `spirv-std` in `Cargo.toml`");
```

Comment:
Neat! I suspect you haven't tested this because of the syntax error. If you don't want to escape the double quotes here you can use a raw string, e.g.:
```suggestion
compile_error!(
    r#"`spriv-std` now requires the use of `glam`. Make sure the "glam" feature is specified for `spirv-std` in `Cargo.toml`"#
);
```

Reply:
Typical test - make last minute change to error message - commit 🙄